documentation/contributing/index.markdown

```diff
@@ -0,0 +1,108 @@
+
+To edit or add more tutorials, create Github pull requests to the appropriate location and the maintainers will graciously approve your help after review.
+
+If you are interested in generating the html pages locally on your machine, install [rosdoc_lite](http://wiki.ros.org/rosdoc_lite) (``apt-get install ros-kinetic-rosdoc-lite``) and run in the root of the package:
+
+    rosdoc_lite .
+
+i.e. [pr2_moveit_tutorials](https://github.com/ros-planning/moveit_pr2/tree/indigo-devel/pr2_moveit_tutorials). This will automatically pick up the rosdoc.yaml and write the documentation to ``doc/html``
+
+For deploying documentation changes to the web, note [Section 3 of rosdoc_lite wiki](http://wiki.ros.org/rosdoc_lite):
+
+>  rosdoc_lite is automatically run for packages in repositories that have rosinstall files listed in the rosdistro repository. The resulting documentation is uploaded to ros.org and is linked in the "Code API" links that you see on various package pages, like rospy.
+
+This is done about once every 24 hours, [overnight](http://wiki.ros.org/rosdistro/Tutorials/Indexing%20Your%20ROS%20Repository%20for%20Documentation%20Generation).
+
+### moveit.ros.org
+
+Once you have created new Sphinx documentation, you'll need to create links to it from the main moveit.ros.org website so that other users can find it. Additionally, some higher level documentation can be found directly on this website, so you'll need to edit it there as well.
+
+To edit or add pages on the MoveIt! website, go to [https://github.com/ros-planning/moveit.ros.org](https://github.com/ros-planning/moveit.ros.org) and either edit inline using Github's web-based editing capability, or ``git clone`` the repo and push pull requests back to the repo.
+
+## Answering User Questions
+
+You can contribute greatly to the MoveIt! community by answer other people's questions on the [MoveIt! mailing list](https://groups.google.com/forum/#!forum/moveit-users)
+
+Additionally, the ROS community asks questions pertaining to MoveIt on [answers.ros.org](http://answers.ros.org/questions/scope:all/sort:activity-desc/tags:moveit/page:1/). Please feel free help answer questions there, and gain Karma in the process!
+
+As an aside, a lot of documentation exists as Q&As on there, and if you feel so compelled, help us move some of that documentation onto the website.
+
+## Reporting & Fixing Bugs
+
+Bugs should be reported to the Github issues page of the appropriate repo. Please leave as much detail as possible including ROS and Ubuntu version. Note however that we have limited resources to fix individual bugs, and are more inclined to give advice on helping you fix your own bugs.
```

Comment:
I've changed it to read:

> including your ROS version, operating system, operating system version, and whether you built from source

I think a better place to the greater details that @130s is referring to is in an issues template: https://github.com/blog/2111-issue-and-pull-request-templates

This would be great to have but annoying duplicating it across ~16 moveit repos, so I say we hold off on it...